### PR TITLE
(msg_hash_us.h) Small touch up

### PR DESCRIPTION
**Pull request by GABO1423**
## Description

This takes care of a FIXME comment in the file. Shouldn't affect much else, but reviews are still welcomed. 

Fix by @Gamr13

### CODE REVIEW

File: menu/menu_displaylist.c [hunk 1]
```diff
@@ -1696,6 +1696,7 @@ static unsigned menu_displaylist_parse_system_info(file_list_t *list)
             cpu_arch_str, sizeof(cpu_arch_str));
 
       strlcpy(cpu_str, cpu_text_str, sizeof(cpu_str));
+      strlcat(cpu_str, ": ", sizeof(cpu_str));
       strlcat(cpu_str, " ", sizeof(cpu_str));
```
Comment (Cthulhu-throwaway) [inline menu/menu_displaylist.c:1700]: This line needs to be removed.

Comment (Gamr13) [reply menu/menu_displaylist.c:1700]: Line 1700 with the blank " "?

Comment (Cthulhu-throwaway) [reply menu/menu_displaylist.c:1700]: Yes. Space is already added by the new line. If you don't remove that line, you've a colon followed by two whitespaces.

Comment (Gamr13) [reply menu/menu_displaylist.c:1700]: Grand, will do. Cheers!

Comment (GABO1423) [reply menu/menu_displaylist.c]: Line has been removed

**Pull request merged**